pkgs/applications/misc/polychromatic/default.nix

```
@@ -48,9 +48,11 @@ python3Packages.buildPythonApplication rec {
     ninja
     sassc
     wrapGAppsHook
+    qt5.wrapQtAppsHook
   ];
 
   propagatedBuildInputs = with python3Packages; [
+    libxcb
     colorama
     colour
     gtk3
```

```suggestion
    colorama
    colour
    gtk3
```
We probably only need libxcb in buildInputs since it will be in the wrapper and not in PYTHONPATH

---

I tested this and it works without it under the propagatedBuildInputs, as long as we have dontWrapQtApps = false;